furious/processors.py

```diff
@@ -67,7 +67,7 @@ def run_job():
     except AbortAndRestart as restart:
         logging.info('Async job was aborted and restarted: %r', restart)
         raise
-    except Exception as e:
+    except BaseException as e:
```

Comment:
Hmm, so Google is inheriting from `BaseException` instead of `Exception`?

That's not cool. https://docs.python.org/2/library/exceptions.html#exceptions.BaseException